src/backend/TrafficCourts/Citizen.Service/Services/Tickets/Search/TicketSearchService.cs

```diff
@@ -134,28 +134,40 @@ private Models.Tickets.ViolationTicket AssembleViolationTicket(string ticketNumb
 
     public async Task<bool> IsDisputeSubmittedBefore(string ticketNumber, CancellationToken cancellationToken)
     {
-        // Check if a dispute has been submitted before for the given ticket number by verfying any Dispute exists associated to the provided ticket number
-        SearchDisputeRequest searchRequest = new() { TicketNumber = ticketNumber };
-        Response<SearchDisputeResponse> response = await _bus.Request<SearchDisputeRequest, SearchDisputeResponse>(searchRequest, cancellationToken);
+        try
+        {
+            // Check if a dispute has been submitted before for the given ticket number by verfying any Dispute exists associated to the provided ticket number
+            SearchDisputeRequest searchRequest = new() { TicketNumber = ticketNumber, ExcludeStatus = ExcludeStatus2.REJECTED };
+            Response<SearchDisputeResponse> response = await _bus.Request<SearchDisputeRequest, SearchDisputeResponse>(searchRequest, cancellationToken);
 
-        var searchResponse = response.Message;
+            var searchResponse = response.Message;
```

Comment:
extra tab?

Reply:
I just re-committed that line but still looks the same. It's the formatter of my VS editor I think.